laravel/resources/js/admin.js

```diff
@@ -65,15 +66,33 @@ $(document).ready(function()
         var upload = $(this).parents('.upload').data('id');
         var status = $(this).parents('.upload').find('.upload-status').value();
         var csrf = $('.csrf-token').value();
+        var slot = $('.slot-number').value(); //store var here
         var data =
         {
             status: status,
             _token: csrf
         };
 
         ajaxOptions.body = JSON.stringify(data);
-        fetch('/upload/' + upload + '/edit', ajaxOptions);
+        
+        console.log(upload,status);
+
+        // if there is nothing to upload, then check for volunteers who flaked
+        if (!upload) {
+            console.log('slot number: '+ slot);
+            console.log(fetch('/slot/'+ slot +'/edit',ajaxOptions)); 
```

Comment:
Editing a slot has nothing to do with file uploads. The code to edit flake statuses should be completely separate from file uploads.